expression/builtin_time_test.go

```diff
@@ -856,7 +856,7 @@ func TestNowAndUTCTimestamp(t *testing.T) {
 		require.NoError(t, err)
 		mt = v.GetMysqlTime()
 		require.True(t, strings.Contains(mt.String(), "."))
-		require.LessOrEqual(t, ts.Sub(gotime(mt, ts.Location())), 3*time.Second)
```

Comment:
The test check call `time.Now()`, and then execute the expression function `Now`,
the difference is within 3s... 
Unless the computer is very slow (under extremely high load), this should not takes 3s+

You can just change 3s to 4s or 5s ... if this test case is not stable enough @hawkingrei

Reply:
have set to 5s.